stable/airflow/README.md

```diff
@@ -658,163 +698,4 @@ __Airflow Prometheus Values:__
-
-Two new parameters:
-* `web.baseUrl`
-* `flower.urlPrefix`
-
-__WARNING:__ This upgrade will fail if a custom ingress path is set for web and/or flower and `web.baseUrl` and/or `flower.urlPrefix`
-
-### To 4.0.0
-This version splits the specs for the NodeSelector, Affinity and Toleration features. 
-Instead of being global, and injected in every component, they are now defined _by component_ to provide more flexibility for your deployments. 
-As such, the migration steps are really simple. 
-Just copy and paste your node/affinity/tolerance definitions in the four airflow components, which are `worker`, `scheduler`, `flower` and `web`. 
-The default values file should help you with locating those.
-
-### To 3.0.0
-This version introduces a simplified way of managing secrets, including the database credentials to postgres and redis.
-With the default settings in prior versions, database credentials were generated and stored in an Airflow-managed Kubernetes secret.
-However, these credentials were also stored in postgres- and redis-managed secrets (created by the respective subcharts), leading to duplication.
-Moreover, it was tricky to bring your own passwords and to load additional secrets as environment variables.
-
-To deal with these issues, we've removed the Airflow-managed Kubernetes secret (`templates/secret-env.yaml`).
-If your deployment was called `airflow`, this upgrade will delete the `airflow-env` secret.
-Instead, the pods now source the database secrets from the postgres- and redis-managed secrets, i.e. the postgres password is in the `airflow-postgres` secret.
-This upgrade _shouldn't_ break the deployment, but you may need to make some adjustments if you were doing something nonstandard.
-
-For production, it's better create random passwords before installing the Helm chart.
-You can use these passwords by specifying the newly added `postgres.existingSecret` and `redis.existingSecret` parameters.
-
-We've also added `airflow.extraEnv`, which provides a flexible way to inject environment variables into your pods.
-This parameter is great for things like the Fernet key and LDAP password.
-
-The following parameters are no longer necessary and have been removed: `airflow.defaultSecretsMapping`, `airflow.secretsMapping`, `airflow.existingAirflowSecret`.
-If you were using them, you'll have to migrate your settings to `postgres.existingSecret`, `redis.existingSecret`, and `airflow.extraEnv`, which are described in greater depth in the documentation above.
-
-### To 2.8.3
-The parameter `airflow.service.type` no longer applies to the Flower service, but the default of `ClusterIP` has been maintained.  
-If using a custom values file and have changed the service type, also specify `flower.service.type`.
-
-### To 2.0.0
-The parameter `workers.pod.annotations` has been renamed to `workers.podAnnotations`.  
-If using a custom values file, rename this parameter.
+| `prometheusRule.groups` | alerting rules for Prometheus | `[]` |
```

Comment:
Can you keep the "Upgrading" section and only add a link to Changelog.md?

Reply:
done